### PR TITLE
Add Llama 3.1 models

### PR DESCRIPTION
**Pull request by seanshi-scale**
# Pull Request Summary

Add Llama 3.1 models. Note: 405b availability will come separately, as we need multinode for it.

## Test Plan and Usage Guide

Test via trying to deploy models on cluster

### CODE REVIEW

**Review by dmchoiboi — COMMENTED** [review]
missing a few places: https://github.com/scaleapi/llm-engine/pull/572/files#diff-3de965f7f38d04aec3e7eea365ad9f30218afe0bfa90513983f062d6e9340727L196

**Review by dmchoiboi — APPROVED** [review]

File: model-engine/model_engine_server/inference/vllm/vllm_server.py [hunk 1]
```diff
@@ -253,19 +252,20 @@ def extract_logprobs(logprobs: Dict[int, Logprob]) -> Dict[int, float]:
...
 if __name__ == "__main__":
     check_unknown_startup_memory_usage()
 
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--host", type=str, default=None)  # None == IPv4 / IPv6 dualstack
-    parser.add_argument("--port", type=int, default=5005)
-    parser = AsyncEngineArgs.add_cli_args(parser)
-    args = parse_args()
+    parser = FlexibleArgumentParser()
+    # host, port, and AsyncEngineArgs are already given by make_arg_parser() in parse_args()
```
Comment (dmchoiboi) [inline model-engine/model_engine_server/inference/vllm/vllm_server.py:264]: delete?

File: model-engine/model_engine_server/inference/vllm/vllm_server.py [hunk 2]
```diff
@@ -287,11 +287,18 @@ def parse_args():
         model_config,
         served_model_names,
         args.response_role,
-        args.lora_modules,
-        args.chat_template,
+        lora_modules=args.lora_modules,
+        chat_template=args.chat_template,
+        prompt_adapters=args.prompt_adapters,
```
Comment (dmchoiboi) [inline model-engine/model_engine_server/inference/vllm/vllm_server.py:292]: will we need to add these to batch?

Comment (seanshi-scale) [reply model-engine/model_engine_server/inference/vllm/vllm_server.py:292]: yeah we probably need to, will do this

Comment (seanshi-scale) [reply model-engine/model_engine_server/inference/vllm/vllm_server.py:292]: actually I think we don't need to make changes to batch, doesn't look like it asks for anything OAI related

File: model-engine/model_engine_server/inference/vllm/vllm_server.py (hunk 1 above)
Comment (seanshi-scale) [reply model-engine/model_engine_server/inference/vllm/vllm_server.py:264]: deleted code, kept comments in

**Pull request merged**